assets/js/common/ProfileMenu/ProfileMenu.jsx

```diff
@@ -43,12 +44,12 @@ function ProfileMenu({ username, email, logout }) {
           </span>
         </Menu.Item>
         <Menu.Item>
-          <a
+          <Link
```

Comment:
Doesn't `Link` use `to` instead of `href`?

Reply:
yes, I went a bit too fast :sweat_smile: